src/main/java/me/sat7/dynamicshop/commands/SetTax.java

```diff
@@ -36,6 +38,26 @@ static boolean setTax(String[] args, Player player) {
                 player.sendMessage(DynamicShop.dsPrefix + LangUtil.ccLang.get().getString("ERR.WRONG_DATATYPE"));
             }
         }
+        else if (args.length == 4 && args[1].equals("temp")) {
+            try
+            {
+                int newValue = Integer.parseInt(args[2]);
+                int tempTaxDurationMinutes = Integer.parseInt(args[3]);
+                if(newValue <= 2) newValue = 2;
+                if(newValue > 99) newValue = 99;
+                if(tempTaxDurationMinutes <= 1) tempTaxDurationMinutes = 1;
+
+                ConfigUtil.setCurrentTax(newValue);
+                Bukkit.getScheduler().runTaskLater(DynamicShop.plugin, ConfigUtil::resetTax, 20L * 60L * tempTaxDurationMinutes);
```

Comment:
What happens if the user uses this command again before the timer expires? maybe you should cancel old task.

Reply:
Good catch. Will fix this